orchagent/muxorch.cpp

```diff
@@ -1229,10 +1264,33 @@ bool MuxOrch::handlePeerSwitch(const Request& request)
                            MUX_TUNNEL, peer_ip.to_string().c_str());
             return false;
         }
-
         auto it =  dst_ips.getIpAddresses().begin();
         const IpAddress& dst_ip = *it;
-        mux_tunnel_id_ = create_tunnel(&peer_ip, &dst_ip);
+
+        // Read dscp_mode of MuxTunnel0 from decap_orch
+        string dscp_mode_name = decap_orch_->getDscpMode(MUX_TUNNEL);
+        if (dscp_mode_name == "")
+        {
+            SWSS_LOG_INFO("dscp_mode for tunnel %s is not available yet", MUX_TUNNEL);
+            return false;
```

Comment:
We do not have to check for these new attributes. The dst_ips get will ensure MUX_TUNNEL is created. We can have these attributes as optional. We don't have to worry about test failures if the MUX_TUNNEL is not populated right away.

Reply:
Fixed. Thanks